packages/components/src/range-control/index.js

```diff
@@ -55,7 +55,7 @@ function RangeControl(
 		onFocus = noop,
 		onMouseMove = noop,
 		onMouseLeave = noop,
-		railColor,
+		railColor = 'var( --wp-admin-theme-color )',
```

Comment:
Not introduced by this PR, but I got a bit confused with the `rail` vs `track` naming. Do they refer to the same part of the UI ?

The README even has an `Anatomy` part where it specifically highlights the "Track", but then there is a prop called `railColor`, a file called `rail.js` and, in the styles, there are CSS snippets refererrin to both the track and the rail.

Should we try to use only one term everywhere?

Reply:
Rail is the "filled in" side and "track" is the background.

Reply:
Should we add this explanation to the README, maybe adding "Rail" to the "Anatomy" section?